src/customize-pop/keyboard-settings.md

```diff
@@ -0,0 +1,70 @@
+# Keyboard Settings
+
+Option to set input Sources, input source switching, special character entry options, and keyboard shortcuts are located in `Settings` ➞ `Keyboard`.
+
+![Keyboard Settings](/images/keyboard-settings/keyboard-settings.png)
+
+| Option | Function |
+|--------|----------|
+| Input Sources | Choose multiple input languages and layouts for a single keyboard. Sources can be moved up or down to determine priority. |
+| Input Source Switching | Set a keyboard shortcut for switching between your chosen sources. |
+| Special Character Key | Designate keys that enable entry of special characters. |
+| Keyboard Shortcuts | View, edit, and add keyboard shortcuts. |
+
+## Adding an Input Source
+
+1. Click the `+` to display available input sources. Select an input source from the list, or click the three vertical dots to perform a keyword search.
```

Comment:
> 1. Select an input source from the list,
> 2. Select a language from the list.

Selecting an item from the list should not need to be repeated in two steps. I would recommend removing the second sentence from the first step. (If it's referring to selecting an already-configured input source, then it still shouldn't be here, because that's not what this section is about.)

Reply:
Fixed